index.js

```diff
@@ -0,0 +1,74 @@
+function displayData(articles) {
+  const stories = document.querySelector("#article");
+  articles.map(article => {
```

Comment:
If you don't use the output of a map method, it's better to replace it with a forEach instead